picard/tagger.py

```diff
@@ -656,45 +656,42 @@ def sighandler(self):
         self.signalnotifier.setEnabled(True)
 
 
-def help():
-    print("""Usage: %s [OPTIONS] [FILE] [FILE] ...
-
-If one of the filenames begins with a hyphen, use -- to separate the options
-from the filenames.
-
-Options:
-    -d, --debug             enable debug-level logging
-    -h, --help              display this help and exit
-    -v, --version           display version information and exit
-    -V, --long-version      display long version information and exit
-""" % (sys.argv[0],))
-
-
 def version():
     print("%s %s %s" % (PICARD_ORG_NAME, PICARD_APP_NAME, PICARD_FANCY_VERSION_STR))
 
+
 def longversion():
     print(versions.as_string())
 
 
+def process_picard_args():
+    parser = argparse.ArgumentParser(
+        epilog="If one of the filenames begins with a hyphen, use -- to separate the options from the filenames."
+    )
+    parser.add_argument("-d", "--debug", action='store_true',
+                        help="enable debug-level logging")
+    parser.add_argument('-v', '--version', action='store_true',
+                        help="display version information and exit")
+    parser.add_argument("-V", "--long-version", action='store_true',
+                        help="display long version information and exit")
+    parser.add_argument('FILE', nargs='*')
```

Comment:
How does this work exactly? Does it capture only the arguments without a leading `-` or would it also capture other unknown arguments?